cmd/explaintest/run-tests.sh

```diff
@@ -17,16 +17,13 @@ TIDB_TEST_STORE_NAME=$TIDB_TEST_STORE_NAME
 TIKV_PATH=$TIKV_PATH
```

Comment:
Bazel's ```sh_test``` can be used here.

Reply:
OK, I will do it later with bazel support.